### ISSUE TITLE
Investigate/Fix bbc-a11y on local

### ISSUE THREAD

**Issue by EinsteinNjoroge** [bbc/simorgh#3200]
**Describe the bug**
A11y tests for igbo, pidgin, thai and yoruba services are failing on local.

**To Reproduce**
1. Switch to `make-a11y-test-more-dynamic` branch
2. run `npm run bbcA11y`

A11y tests fail on local

**Expected behavior**
A11y test should pass on local

**Screenshots**
<img width="611" alt="Screen Shot 2019-08-16 at 3 15 32 PM" src="https://user-images.githubusercontent.com/21374761/63169254-f73d3600-c03e-11e9-98be-1e317bccfe7d.png">

**Desktop (please complete the following information):**

- OS: MacOS (Only)

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

**Comment by EinsteinNjoroge** [comment]
Closing this issue in favor of  https://github.com/bbc/bbc-a11y/issues/296
